box-tests.sh

```diff
@@ -28,11 +21,17 @@ runtests35-sqlite3 auth_tests
 runtests35-postgres auth_tests
 runtests35-mysql auth_tests
 
-runtests27-sqlite3 admin_widgets --selenium chrome
+runtests36-sqlite3-gis gis_tests
```

Comment:
It may help to use the same terminology as Jenkins: spatialite, postgis, mysql_gis